tests/rpc/calls/getCode.test.ts

```diff
@@ -0,0 +1,66 @@
+import { getCodeHandler } from '../../../src/rpc/calls/getCode'
+import { RPCResponse } from '../../../src/types/types'
+
+describe('Test getCodeHandler', () => {
+  it('Returns class hash for a valid request', async () => {
+    const request = {
+      jsonrpc: '2.0',
+      method: 'eth_getCode',
+      params: ['0xce77ddf883b3e92cddd6675c7553edd30d1cf64a', 960987],
+      id: 0,
+    }
+
+    const starknetRes: RPCResponse = <RPCResponse>await getCodeHandler(request)
+
+    expect(typeof starknetRes.result).toBe('string')
+    expect(starknetRes.result).toBe(
+      '0x29927c8af6bccf3f6fda035981e765a7bdbf18a2dc0d630494f8758aa908e2b',
+    )
+  })
+
+  it('Returns empty result for non-existant eth address', async () => {
+    const request = {
+      jsonrpc: '2.0',
+      method: 'eth_getCode',
+      params: ['0x94750381bE1AbA0504C666ee1DB118F68f0780D4', 960987],
+      id: 0,
+    }
+
+    const starknetRes: RPCResponse = <RPCResponse>await getCodeHandler(request)
+
+    expect(typeof starknetRes.result).toBeNull
```

Comment:
This approach isn't correct. You're putting the `toBeNull` method over the `expect()` block.
So, even if you remove the null check, i.e.,
```
expect(typeof starknetRes.result)
expect(starknetRes.result)
```
Tests still pass :)

Instead, check if the result is null, or better yet, you can match the entire result object like this
```
expect(starknetRes).toMatchObject({
  jsonrpc: '2.0',
  id: 0,
  result: undefined,
});
```